root/static/scripts/edit/MB/CoverArt.js

```diff
@@ -410,7 +410,8 @@ MB.CoverArt.FileUpload = function (file) {
                     self.updateProgress(2, 100);
 
                     var submitting = MB.CoverArt.submit_edit(
-                        self, postfields, mime_type, position);
+                        self, postfields, mime_type, position
```

Comment:
Should i add new line after each comma here?

Reply:
So that it is consistent with other files changed.

Reply:
Well, I would. :) I don't think we have a rule for it, but I find it easier to read that way.

Reply:
okay thanks